client/my-sites/email/email-management/home/email-plan-warnings.jsx

```diff
@@ -56,45 +57,57 @@ class EmailPlanWarnings extends React.Component {
 		);
 	}
 
-	renderWarningCTA( warning ) {
-		const warningSlug = warning.warning_slug;
-		if ( ! warningSlug ) {
-			return null;
-		}
+	renderCTAForGooglePendingTOSAcceptance() {
+		const { domain, translate } = this.props;
+
+		return (
+			<Button compact primary href={ getGoogleAdminUrl( domain.name ) } target="_blank">
```

Comment:
We may want to adjust this URL in favour of the `getGoogleAdminWithTosUrl()` function [flagged in this comment](https://github.com/Automattic/wp-calypso/pull/53032#pullrequestreview-664396027).

It may make sense to keep this logic as-is unless we land #53032 first.